examples/docker-mixin-test/porter.yaml

```diff
@@ -4,71 +4,54 @@ description: "An example bundle that uses docker through the magic of whalespeak
 tag: gmadhok/porter-hello/whalesay:v0.1.1
```

Comment:
Minor nit on the bundle name being `whalesay`, the same as the example bundle in the porter repo using the docker-compose mixin.  (On the off-chance that I've generated parameter sets for both bundles, the last one set would overwrite the other, owing to the same name... which, arguably, is a Porter bug 😉 )  Maybe a slightly different name for this example bundle?

Reply:
Got it I can change the name!